projects/angular-calendar/src/modules/month/calendar-month-row.component.ts

```diff
@@ -149,12 +153,17 @@ export class CalendarMonthRowComponent implements OnChanges {
       .map((daySliced) =>
         this.timezoneManager.reverseTz(daySliced.date, this.timezone)
       )
-      .findIndex((date) => date.valueOf() === note.start.valueOf());
+      .findIndex(
+        (date) =>
+          new Date(date.toDateString()).valueOf() === note.start.valueOf()
```

Comment:
date.toDateString() === note.start.toDateString() is better no ?

Reply:
No need as note.start is already timezone free so note.start is equal to new Date(note.start.toDateString())